src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/SetAzureRmDataLakeStoreItemAclEntry.cs

```diff
@@ -19,11 +19,12 @@
 using System.Linq;
 using System.Management.Automation;
 using Microsoft.Azure.DataLake.Store.Acl;
+using Microsoft.Azure.DataLake.Store.AclTools;
 
 namespace Microsoft.Azure.Commands.DataLakeStore
 {
     [Cmdlet(VerbsCommon.Set, "AzureRmDataLakeStoreItemAclEntry", SupportsShouldProcess = true, DefaultParameterSetName = BaseParameterSetName),
-     OutputType(typeof(bool))]
+     OutputType(typeof(IEnumerable<DataLakeStoreItemAce>))]
```

Comment:
This should just be DataLakeStoreItemAce - powershell treats single objects and enumerations of objects as identical.

Reply:
@markcowl Fixed.